src/typing_extensions.py

```diff
@@ -1819,6 +1830,10 @@ class Movie(TypedDict):
 
 if hasattr(typing, "Unpack"):  # 3.11+
     Unpack = typing.Unpack
+
+    def _is_unpack(obj):
+        return get_origin(obj) is Unpack
```

Comment:
This change was required in order for tests to pass on Python 3.11. Without this change, a function somewhere was failing with `NameError` when the tests were run on Python 3.11, because the function was calling `_is_unpack()`, and we previously only defined `_is_unpack` on Python <=3.10.